src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx

```diff
@@ -78,24 +94,25 @@ The diagram below shows the flow of data with the collector.
 To use the collector:
 
 1. Configure your OpenTelemetry collector to export data to New Relic, using our [example](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/newrelicexporter) as a guide.
-2. Configure your service’s OTLP exporter to send data to your collector, following the documentation for your language's OTLP exporter:
-   * [C++](https://github.com/open-telemetry/opentelemetry-cpp/tree/master/exporters/otlp)
-   * [Erlang](https://github.com/open-telemetry/opentelemetry-erlang/tree/master/apps/opentelemetry_exporter)
-   * [Go](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp)
-   * [Java](http://github.com/open-telemetry/opentelemetry-java-instrumentation#getting-started)
-   * [Javascript/Node.js](https://www.npmjs.com/package/@opentelemetry/exporter-collector)
-   * [.NET](https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/src/OpenTelemetry.Exporter.OpenTelemetryProtocol)
-   * [PHP](https://github.com/open-telemetry/opentelemetry-php/blob/master/examples/AlwaysOnOTLPExample.php)
-   * [Python](https://opentelemetry-python.readthedocs.io/en/stable/exporter/otlp/otlp.html)
-   * [Ruby](https://github.com/open-telemetry/opentelemetry-ruby/tree/master/exporter/otlp)
-   * [Rust](https://crates.io/crates/opentelemetry-otlp)
-   * [Swift](https://github.com/open-telemetry/opentelemetry-swift/tree/master/Sources/Exporters/OpenTelemetryProtocol)
-   * [...Find additional OTLP language support in GitHub](https://github.com/open-telemetry)
+
+<Callout variant="important">
+The New Relic exporter for the collector will be deprecated in the future. If you are running your own collector, you will instead be able to use the [OpenTelemetry Collector's OTLP exporter]() When we release our [native OTLP endpoint (pre-release)](#otlp), th If you were previously using a New Relic language-specific exporter consider using the OTLP exporter for your language and send data directly to New Relic's [native OTLP endpoint (pre-release)](#otlp).
```

Comment:
Hi @alanwest - there's a link missing here.

Reply:
@urbiz-nr yep, that's the link, thanks!